### DISCUSSION TITLE
Menu's Staying opening till click action

### DISCUSSION THREAD

**Discussion by ChasCB** [Aris-t2/CustomCSSforFx#605] [Q&A] [answered]
Since either FF113 or FF114 I have found that when I click on either the Application Menu or the Bookmarks overflow menu or Add-ons from within the NavBar (ie 1Password), they stay open till I click on a selection. Even if I click away from (out of) onto the browser page they still stay open.
I thought they used to close when you clicked out of them or they timed out and closed automatically. As you can see you can even have multiple overlapped menus.

Very annoying.

![Screen Shot 2023-06-09 at 15 03 48](https://github.com/Aris-t2/CustomCSSforFx/assets/7608263/e38ecab0-c1f2-41e0-9dfc-17362a54363f)

**Comment by ChasCB** [comment, marked as answer]
Mystery solved. Though I have not been in Firefox about:config for months the 'ui.popup.disable_autohide' setting was set to 'true' when it should have been 'false'. Pop-ups now, again, close when you click outside of them.

**Comment by Aris-t2** [comment]
Good to know you could solve this.
